### ISSUE TITLE
Force loading objects

### ISSUE THREAD

**Issue by andrzejewsky** [vuestorefront/vue-storefront#5671]
At this moment, when we call `load` function from our composables, the given object is being loaded only once. That was intentional to protect the front-end from overloading (loading the same object many times). 

Sometimes there is a need to load the objects anyway, even if we already have loaded them. To solve the problem I suggest adding `cacheStrategy` to each composable:

```
const { products } = useProduct({ key: '', cacheStarategy: 'memory' })

cacheStarategy: string | cacheDriver

cacheDriver: store (from https://www.npmjs.com/package/axios-cache-adapter )
```

Note previous syntax should still work as well `useProduct('key')`

**Managing cache strategies should be handled in the core on the frontend side**

Moreover we should be able to specify default cache strategy for composables in config of vsf nuxt module

    [`@vue-storefront/nuxt`, {
      clientCache: {
        driver: string | cacheDriver (built-in or custom)
      }
    }]
    
    eg
    
    [`@vue-storefront/nuxt`, {
      clientCache: {
        driver: `memory`
      }
    }]There should  be a possibility to write your own Cache Driver. [~accountid:5fcf8703b4a74900762889bf] we should decide if that should be made through axios or in the core factories (I suggest [axios](https://www.npmjs.com/package/axios-cache-adapter))To discuss: How would this relate to SW Cache?

Perhspas we need just Workbox

**Comment by filipsobol** [comment]
This is implemented only in `useCart` factory. We should consider whether we should implement this in other factories as well, or if it's needed at all in `useCart` and could be removed.

If we decide to implement it in other factories, we should consider throttling composable methods, eg. if the given method was already called and we are waiting for the response or if the response came less than 1 second ago, we could skip the API requests and return the previous result immediately. This would work great when the page is Server Side Rendered and few components call the same method in a very short time, but wouldn't happen later when the user interacts with the application.

Additionally, as you mentioned, we can introduce `reset`, `force` or `refresh` parameter to avoid throttling in some specific cases.

**Comment by andrzejewsky** [comment]
@filipsobol would be nice to have it across all of the factories where we have a load function that loads objects only once, but let's start with a cart as first.

**Comment by filrak** [comment]
opt for `reload`
